birdhouse/components/jupyterhub/default.env

```diff
@@ -80,16 +80,24 @@ export JUPYTERHUB_ADMIN_USERS='{\"${MAGPIE_ADMIN_USERNAME}\"}'  # python set syn
 #  export JUPYTERHUB_RESOURCE_LIMITS="
 #     user:user1:mem_limit=30G
 #     group:group1:mem_limit=10G:cpu_limit=1
-#     group:group2:cpu_limit=3
+#     group:group2:cpu_limit=3:gpu_ids=gpu1,gpu2,gpu3
+#     user:user2:gpu_ids=gpu1,gpu2,gpu3:gpu_count=2
 #  "
 #
-#  Supported limits are: `mem_limit` and `cpu_limit`. See the Jupyterhub Dockerspawner documentation
-#  for details and supported values.
+#  Supported limits are: `mem_limit`, `cpu_limit`, `gpu_count`, `gpu_ids`. See the Jupyterhub Dockerspawner documentation
+#  for details and supported values for mem_limit and cpu_limit.
+#   - https://jupyterhub-dockerspawner.readthedocs.io/en/latest/api/index.html#dockerspawner.DockerSpawner.cpu_limit
+#   - https://jupyterhub-dockerspawner.readthedocs.io/en/latest/api/index.html#dockerspawner.DockerSpawner.mem_limit
+#  gpu_ids are a comma separated list of the GPU ids available on the host that you want to make available to
+#  the user or group. GPU ids can typically be discovered by running the `nvidia-smi` command.
+#  If gpu_count is also specified, this is an integer indicating how many GPUs to make available to that user or group.
+#  For example, if gpu_ids=gpu1,gpu2,gpu6 and gpu_count=2 then two GPUs will be randomly selected from the gpu_ids list.
```

Comment:
If omitted, is it default 1 or "all"?

I personally think 1 would be safer for fair/shared-use and avoid over allocating, but the default should be indicated either way.

Reply:
Following https://github.com/bird-house/birdhouse-deploy/pull/616#discussion_r2609131646, I see also that group allocations could be misinterpreted easily if `gpu_count` is omitted. The same problem would happen of a single user holding all resources offered to the group.

For groups-based allocations, I think the typical use-case would more often be to provide many GPUs to meet demands, but still distribute/limit them across their users.